javalib/src/main/scala/java/nio/channels/FileChannelImpl.scala

```diff
@@ -57,22 +57,40 @@ final class FileChannelImpl(path: Path,
                     start: Int,
                     number: Int): Long = {
     ensureOpen()
-    val dst = new Array[Byte](1)
-    val nb  = raf.read(dst)
-    var i   = 0
-    while (i < number) {
-      buffers(start + i).put(dst)
-      i += 1
+
+    var bread: Long = 0
```

Comment:
Ambiguous name wrt 🥖.

Reply:
Yeah, I thought it'd be a fun way to write bytesRead in way that enforces the correct reading of read (like "red", as opposed to "reed"). But I can change it.

Reply:
I looked at that too and said "what!". Then I figured maybe it was `bytesRead` but given your other short names it seemed out of place. I liked the 🍞 icon though. 😃